modules/gcloud/gcloud.go

```diff
@@ -68,6 +70,20 @@ func WithProjectID(projectID string) Option {
 	}
 }
 
+// WithDataYamlFile seeds the Bigquery project for the GCloud container.
+func WithDataYamlFile(dataYamlFile string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
```

Comment:
Since #2267 all customiser options must return error, could you update that? [This line](https://github.com/testcontainers/testcontainers-go/pull/2521/files#diff-8faaed968761085a8ca6d01c0502d2c6159fabf32bec52d69cd76a86e56ecb87R28) will need to handle the errors 🙏